### ISSUE TITLE
Can't permanently delete soft-deleted items

### ISSUE THREAD

**Issue by Bonapara** [twentyhq/core-team-issues#110]
# Bug description 

https://github.com/user-attachments/assets/c73a9ba9-da9f-4265-b0c4-195cb16a07dd

Can't permanently delete soft-deleted items. This seems to cause a significant lag afterward as well.

**Comment by Bonapara** [comment]
@charlesBochet you may want to have a look

**Comment by charlesBochet** [comment]
Yes, @Weiko has seen it.
@Weiko assigning it to you

**Comment by FelixMalfait** [comment]
Also tracked here fyi: twentyhq/core-team-issues#114

**Comment by FelixMalfait** [comment]
@bosiraphael maybe you can add this to your backlog since it's related to the action bar? (show different action to permanently delete instead of soft delete based on context). Thanks!

**Comment by charlesBochet** [comment]
This should be done, double check on production

**Comment by prastoin** [comment]
## Update
Unless I'm mistaken I can't find the destroy call to action for either both multiple or single record deletion in production. ( tested in two workspaces with default record and custom record )
On `main` I can see a destroy CTA for single selection only ?

Maybe I'm doing the recipe wrong @charlesBochet ?

![Image](https://github.com/user-attachments/assets/06ca426c-226f-493d-bdc0-af986a25da63)

**Comment by charlesBochet** [comment]
We might not have implemented it, could you take it in your current sprint? Should be quite straight forward as we already have the delete many implementation and the destroy one.

**Comment by prastoin** [comment]
Yep no problem, I just would like to clarify if there's any outgoing regression as it might change the way I look forward
Especially on the ~~delete~~ destroy one in production

**Comment by charlesBochet** [comment]
I think I've never implemented it actually: https://github.com/twentyhq/twenty/blob/main/packages/twenty-front/src/modules/action-menu/actions/record-actions/constants/DefaultActionsConfigV2.ts

**Comment by prastoin** [comment]
Unless I'm mistaken the delete one record seems to be implemented in the file you shared L94 https://github.com/twentyhq/twenty/blob/main/packages/twenty-front/src/modules/action-menu/actions/record-actions/constants/DefaultActionsConfigV2.ts#L94

**Comment by prastoin** [comment]
Which one, Multiple or single ?
As Unless i'm mistaken the destroy one record seems to be implemented line 149 in the file you shared ?
https://github.com/twentyhq/twenty/blob/main/packages/twenty-front/src/modules/action-menu/actions/record-actions/constants/DefaultActionsConfigV2.ts#L149
